tests/CoreFX/TopN.CoreFX.Unix.issues.json

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
Can these files have either (1) a full description at the top of the file format, as a comment, or (2) a link to documentation for this format?